debug/targets.py

```diff
@@ -138,6 +138,9 @@ class Target:
     # Supports controlling hart availability through DMCUSTOM.
     support_unavailable_control = False
 
+    # When true it indicates that the hardware limits icount set to 1
+    limits_icount_to_one = False
+
```

Comment:
Instead of `limits_icount_to_one = False` how about `icount_limit = 4` (or whatever default seems reasonable)?

Reply:
I have changed it according to your opinion
.